augur/tasks/util/collection_util.py

```diff
@@ -213,6 +213,25 @@ def get_enabled_phase_names_from_config():
 
     return enabled_phase_names
 
+def get_collection_intervals_from_config():
+    """Get collection intervals from config with fallback to hardcoded defaults."""
+    try:
+        tasks_config = get_section("Tasks")
+        return {
+            'core': tasks_config.get('core_collection_interval', 15),
+            'secondary': tasks_config.get('secondary_collection_interval', 10), 
+            'facade': tasks_config.get('facade_collection_interval', 10),
+            'ml': tasks_config.get('ml_collection_interval', 40)
+        }
```

Comment:
Im a little mixed on whether we need a dict for this overall since its one additional place that needs modifying if we add another worker for some reason.

Also this function seems to be injecting its own default values independently of the config classes that already exist. IMO the defaults should be handled in one place